### PR TITLE
feat: kresify part (1/x)

### PR DESCRIPTION
**Pull request by frezbo**
Kresify, start with Makefile and keep it compatible with drone/dockerfile.

### CODE REVIEW

File: .kres.yaml [hunk 1]
```diff
@@ -0,0 +1,634 @@
+---
+kind: golang.Toolchain
+spec:
+  # image: ghcr.io/siderolabs/tools:v1.8.0-alpha.0
+  buildTags:
+    - tcell_minimal
+    - grpcnotrace
+  makefile:
+    extraVariables:
+      - name: NAME
```
Comment (smira) [inline .kres.yaml:10]: I think this is a bit of a disaster, we're basically rewriting `Makefile` as `.kres.yaml`, 

Can we take a step back and try to analyze what can be done better if we can do that? Or at least analyze some options?

**Comment by smira** [comment]
@frezbo should we close this?

**Comment by frezbo** [comment]
> @frezbo should we close this?

Yes, closing

**Pull request closed without merging**